WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardViewModel.swift

```diff
@@ -146,10 +146,12 @@ final class BlazeCampaignDashboardViewModel: ObservableObject {
     }
 
     func didSelectCampaignList() {
+        analytics.track(event: .DynamicDashboard.dashboardCardInteracted(type: .blaze))
```

Comment:
❓ This method doesn't seem to be triggered from my testing. Is it redundant?

Reply:
You can see these methods being triggered in the attached video. 

https://github.com/woocommerce/woocommerce-ios/assets/524475/baf9e604-a1b7-4e74-b71f-8c3245ac6d7f

This is triggered upon tapping the "View all campaigns" button. 

![image](https://github.com/woocommerce/woocommerce-ios/assets/524475/f088b130-e775-4701-9c1e-91c5cf87bad9)

Reply:
Oh that's true! But I see that tapping on View all campaigns also triggers `configureBlazeSection` in `DashboardViewHostingController`, and both are tracking the same event. Should we remove one of these?